src/components/Popconfirm/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Popconfirm as AntPopconfirm } from "antd";
+
+export const Popconfirm: React.FC<React.ComponentProps<
+  typeof AntPopconfirm
+>> = ({ children, ...props }) => (
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+  <span
```

Comment:
I think a div is better suited for this since divs are used to wrap content where as spans are better for small snippets of text. https://www.tutorialspoint.com/What-is-the-difference-between-HTML-tags-div-and-span#:~:text=Both%20and%20%3Cspan,level%20portion%20of%20a%20document.&text=The%20div%20should%20be%20used,of%20text%2C%20images%2C%20etc.

Reply:
I think this is a unique case because the Span exists solely to stop event propagation. I also want to avoid starting our AntPopconfirms on a new line with a div since they are usually wrapping buttons. That would likely force us to apply additional styles on top of the AntPopconfirm after we import this component.